<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackFibersDialog</class>
 <widget class="QDialog" name="TrackFibersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiber Tracking Opetions</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QComboBox" name="TrackingMethodBox">
     <item>
      <property name="text">
       <string>Deterministic (FACT)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Deterministic (TOD)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Probabilistic (TOD)</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Tract/Seed Voxel</string>
            </property>
            <property name="buddy">
             <cstring>TractsPerSeed</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="TractsPerSeed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>14</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Max Bending Angle</string>
            </property>
            <property name="buddy">
             <cstring>MaxBendAngle</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="MaxBendAngle">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Step Size (mm)</string>
            </property>
            <property name="buddy">
             <cstring>StepSize</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="StepSize">
            <property name="minimum">
             <double>0.500000000000000</double>
            </property>
            <property name="maximum">
             <double>2.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.250000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Min Fiber Length (mm)</string>
            </property>
            <property name="buddy">
             <cstring>MinLength</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="MinLength">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>70</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="SmoothFibers">
            <property name="text">
             <string>Smooth Fibers</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="SmoothQual">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Smoothing Quality</string>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="MetricBox">
       <property name="title">
        <string>Limiting Metric</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="LimitingMetric">
            <item>
             <property name="text">
              <string>FA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>other...</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filePath">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Start Tracking at:</string>
            </property>
            <property name="buddy">
             <cstring>StartTrackingValue</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="StartTrackingValue">
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
            </property>
            <property name="text">
             <string>0.25</string>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Stop Tracking at:</string>
            </property>
            <property name="buddy">
             <cstring>StopTrackingValue</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="StopTrackingValue">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
            </property>
            <property name="text">
             <string>0.25</string>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TrackingMethodBox</tabstop>
  <tabstop>TractsPerSeed</tabstop>
  <tabstop>MaxBendAngle</tabstop>
  <tabstop>StepSize</tabstop>
  <tabstop>MinLength</tabstop>
  <tabstop>SmoothFibers</tabstop>
  <tabstop>MetricBox</tabstop>
  <tabstop>SmoothQual</tabstop>
  <tabstop>LimitingMetric</tabstop>
  <tabstop>StartTrackingValue</tabstop>
  <tabstop>StopTrackingValue</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TrackFibersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SmoothFibers</sender>
   <signal>clicked(bool)</signal>
   <receiver>SmoothQual</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
